### ISSUE TITLE
example of querying a table with contains

### ISSUE THREAD

**Issue by oz123** [msiemens/tinydb#93]
The documentation says I can query tables like a database, however, I am seeing a strange error:
```
 db._db.table('posts').search(Query().filename.contains('markdown'))
*** TypeError: 'Query' object is not callable
```
Can you please add a test case or a concrete example ?

**Comment by msiemens** [comment]
Can you post the imports you've used?

**Comment by oz123** [comment]
Hi, Thanks for the quick response.
Here is the full code, 
https://github.com/oz123/blogit/blob/master/blogit/blogit.py

**Comment by msiemens** [comment]
In which file/line does the error occur? I tired running your test suite, but got a `ImportError: No module named blogit.blogit` error...

**Comment by oz123** [comment]
did you install it with `pip install -e .`? You will not see the error I posted because I went around the problem. I could not use the API you describe.

**Comment by msiemens** [comment]
> You will not see the error I posted because I went around the problem.

Okay, I wanted to see the code that caused the problem including the import statements that were used. Could you point me to the right location?

**Comment by oz123** [comment]
Like I said there is no location. It isn't there. I posted the URI for the code so you will see the full imports.

**Comment by msiemens** [comment]
I assume that the problem is caused by incorrect imports. I asked for the source lines where the problem occurs in order to take a look at the import statements. Do you remember which import statements you used when you had the problem?

**Comment by oz123** [comment]
No. Unfortunately I can't tell you. But I try to reproduce the issue, and if it is an import issue.

**Comment by oz123** [comment]
I can reproduce this error using a clean environment:

```
In [1]: from tinydb import TinyDB, Query
In [2]: db = TinyDB('db.json')
In [3]: User = Query()
In [4]: db.insert({'name': 'John', 'age': 22})
Out[4]: 1

In [5]: db.search(User.name == 'John')
Out[5]: [{u'age': 22, u'name': u'John'}]

In [6]: db.search(User.name.contains('J'))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-6-681d93ef620f> in <module>()
----> 1 db.search(User.name.contains('J'))

TypeError: 'Query' object is not callable
```

Maybe you meant the `regex` is an `re` object? or is it a string like `^J`?

**Comment by msiemens** [comment]
Actually the problem is that `Query.contains` has been renamed to `Query.search` in c9e3d0e62b936c960cf35a36ba3b76a19fdc5d7e, released in v3.0. The reason was that to indicate better that the function works with regexes. Sadly, it looks like I forgot to update the documentation. I've now pushed an update to the docs that reflect this change.

Sorry for the confusion!
